doc/rst/language/spec/interoperability.rst

```diff
@@ -337,6 +356,26 @@ in Chapel you would refer to this ``struct`` via
        var x, y, z: real(64);
      }
 
+Note that the above examples apply for C unions as well, so an example
+for non-typedef'd C ``union`` would be like this:
+
+.. code-block:: chapel
+
+     union Vec3 {
+       double x, y, z;
```

Comment:
This isn't a particularly interesting union... :D

Reply:
Okay, let me think of something better xD

Reply:
It could even be similar to your previous union, simply using a non-typedef'd variant of it...  (to be clear, what's uninteresting about it is that a union whose fields are the same type doesn't have much value in C since... all the fields are the same type, so are redundant).

Reply:
I understand, I'll use the same union then.